strings/reverse_words.py

```diff
@@ -0,0 +1,23 @@
+# Created by sarathkaul on 18/11/19
+
+
+def reverse_words(input_str: str) -> str:
+    """
+    Reverses words in a given string
+    >>> sentence = "I love Python"
+    >>> reverse_words(sentence) == " ".join(sentence.split()[::-1])
+    True
+    >>> reverse_words(sentence)
+    'Python love I'
+    """
+    input_str = input_str.split(" ")
+    new_str = list()
+
+    for a_word in input_str:
+        new_str.insert(0, a_word)
+
+    return " ".join(new_str)
```

Comment:
I just got the idea that `return " ".join(input_str[::-1])` would have probably worked the same way.

Reply:
Line 8

Reply:
I noticed it being used as a doctest, and NOT the actual function. :sweat_smile: Hence, the comment.